Formula/m1ddc.rb

```diff
@@ -0,0 +1,34 @@
+class M1ddc < Formula
+  desc "Control external displays (USB-C/DisplayPort Alt Mode) using DDC/CI on M1 Macs"
+  homepage "https://github.com/waydabber/m1ddc"
+  url "https://github.com/waydabber/m1ddc/archive/175cdf191df15ff4cff1cebef8c07f1d9f49bbf8.tar.gz"
```

Comment:
Is this a stable version?

Reply:
I understand it to be- referencing the docs on [niche (or-self-submitted) stuff](https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff), [the author hasn't noted it as unstable](https://github.com/waydabber/m1ddc/search?q=%22beta%22+OR+%22alpha%22+OR+%22unstable%22&type=issues) and it appears to be maintained + used (the repo has a significant number of stars).

I requested a tagged release from the author at https://github.com/waydabber/m1ddc/issues/21#issuecomment-1374913213 in case declaring this as stable is a hard requirement.

Reply:
yeah, we need a release to rollout for the users.

Reply:
No worries, it seems the author released https://github.com/waydabber/m1ddc/releases/tag/v1.0.0 so I've listed this in https://github.com/Homebrew/homebrew-core/commit/ce76f5bf8cb2014fc1b6c98a0f5ff20a9c1d9066